agent/taskresource/volume/dockervolume_test.go

```diff
@@ -257,7 +257,7 @@ func TestNewVolumeResource(t *testing.T) {
 	}
 
 	for _, testcase := range testCases {
-		t.Run(fmt.Sprintf("%s,scope %s, autoprovision: %s", testcase.description,
+		t.Run(fmt.Sprintf("%s,scope %s, autoprovision: %v", testcase.description,
```

Comment:
The bool verb is `%t`: https://golang.org/pkg/fmt/.  `%v` is fine in a test-case though.

Reply:
huh didn't know that thing was called a "verb", cool.